feedparser/sgml.py

```diff
@@ -27,7 +27,8 @@
 
 import re
 
-import sgmllib
+# FIXME: replace with something else, as sgmllib3k is not maintained. See https://github.com/kurtmckee/feedparser/issues/279
```

Comment:
This is not needed for this patch.

Reply:
I wanted to flag it further as it's lack of updating means there's no mypy work for it at all coming. I've removed it though.